### ISSUE TITLE
Issue with Tezbox Wallet - KT1 account disappeared

### ISSUE THREAD

**Issue by saiasif** [tezbox/web-wallet#62]
Today when I logged into my Tezbox wallet, it displayed a warning once I logged in "This account has not been added to the blockchain yet - please wait for a baker to include this in a block before you can use this account. This error may also show if your device can't connect to the Tezos Network.".  Not sure why this is showing up but on the TZ explorer I can see these are delegated for my KT1 account.

I did some research and it said try restoring your account.  When I did that, now I have two TZ1 accounts but do not see my KT1 account. When I tried adding my KT1 account I am being asked a fee of 0.26TZ which I don't have in my TZ1 account.  Could you please help restore my KT1 account attached to my original TZ1 account.  I am not even seeing an option for Import KT1 account.

**Comment by shaun19826** [comment]
Hello,
Have you been able to resolve your issue ?

**Comment by shaun19826** [comment]
@saiasif  Hello, Have you been able to resolve this issue ?

**Comment by shaun-19826** [comment]
@saiasif Hello, Has your issue been resolved or you're still having this issue ?

**Comment by saiasif** [comment]
@shaun-19826 thanks mate.  Yes finally I was able to get a resolution to my problem.  I found out that my Genesis TZ1 account which is the Manager address is mapped directly to the delegated address KT1 account.  Thus if I am able to restore my TZ1 account, it will automatically restore my KT1 account as well.  As I had all the requisite information in regards to my Genesis account I was able to restore it on Galleon Desktop wallet (https://cryptonomic.tech/galleon.html) and I was able to retrieve all my Tezzies.
